detox/android/detox/src/full/java/com/wix/detox/espresso/action/DetoxAccessibilityAction.java

```diff
@@ -0,0 +1,59 @@
+package com.wix.detox.espresso.action;
```

Comment:
In essence, I have a slight issue with the fact that this RN-specific action resides directly under `espresso` rather than somewhere under a dedicated `react-native` sub-package (e.g. `com.wix.detox.reactnative.espresso`).

That said, I see that we've already gone down that wrong path before - with `RNClickAction`, so at this point it wouldn't be right to ask for relocating things. We'd have to live with the tech debt and postpone that to a later time. Nevertheless, for the time being, I'd appreciate renaming the class to `RNDetoxAccessibilityAction`, so as to make it clear for the future.